controller/todos.go

```diff
@@ -0,0 +1,44 @@
+package controller
+
+import (
+	"github.com/gin-gonic/gin"
+	"godemo/database"
+	"godemo/model"
+	"net/http"
+)
+
+type todos struct{}
+
+var Todos todos = todos{}
```

Comment:
exported var Todos should have comment or be unexported<br>should omit type todos from declaration of var Todos; it will be inferred from the right-hand side